dashboard/app/models/lesson_activity.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: lesson_activities
+#
+#  id         :integer          not null, primary key
+#  lesson_id  :integer          not null
+#  position   :integer          not null
+#  properties :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_lesson_activities_on_lesson_id  (lesson_id)
+#
+
+class LessonActivity < ApplicationRecord
+  include SerializedProperties
+
+  belongs_to :lesson, inverse_of: :activities
+  has_many :activity_sections, inverse_of: :activity, dependent: :destroy
+
+  serialized_attrs %w(
+    display_name
+    time
```

Comment:
@dmcavoy thoughts on renaming this to `duration`, here and in client code? time seems accurate, but not specific.

Reply:
I don't have a strong opinion either way. If you feel like duration is better I think updating it is fine.

Reply:
`duration` does sound better to me, so if you don't feel strongly I will take you up on this and change it.